tools/clang/test/HLSL/workgraph/nodeoutput_array.hlsl

```diff
@@ -8,16 +8,37 @@ struct Record {
 };
 
 [Shader("node")]
-[NodeLaunch("Broadcasting")]
+[NodeLaunch("broadcasting")]
 [NodeMaxDispatchGrid(65535, 1, 1)]
 [NodeIsProgramEntry]
 [NumThreads(32, 1, 1)]
-void node01(NodeOutput<Record> output[9]) // expected-error {{arrays of 'NodeOutput' are not supported as node parameters - use 'NodeOutputArray' instead}}
+// expected-error@+2 {{entry parameter of type 'NodeOutput<Record> [9]' may not be an array}}
+// expected-note@+1 {{'NodeOutput' cannot be used as an array; did you mean 'NodeOutputArray'?}}
+void node01(NodeOutput<Record> output[9])
 { }
 
 [Shader("node")]
-[NodeLaunch("Broadcasting")]
+[NodeLaunch("broadcasting")]
 [NodeMaxDispatchGrid(65535, 1, 1)]
 [NumThreads(32, 1, 1)]
-void node02(EmptyNodeOutput output[4]) // expected-error {{arrays of 'EmptyNodeOutput' are not supported as node parameters - use 'EmptyNodeOutputArray' instead}}
+// expected-error@+2 {{entry parameter of type 'EmptyNodeOutput [4]' may not be an array}}
+// expected-note@+1 {{'EmptyNodeOutput' cannot be used as an array; did you mean 'EmptyNodeOutputArray'?}}
```

Comment:
huh. Didn't know you could do that.

Reply:
The even better trick that you can do in modern clang is put a comment like `// #match-here` on a line then do `//expected-error@match-here {{..}}`. That doesn't work in DXC, but does un upstream and it is super nice.